library/src/main/api21/com/google/android/cameraview/Camera2.java

```diff
@@ -480,8 +713,11 @@ void startCaptureSession() {
         try {
             mPreviewRequestBuilder = mCamera.createCaptureRequest(CameraDevice.TEMPLATE_PREVIEW);
             mPreviewRequestBuilder.addTarget(surface);
-            mCamera.createCaptureSession(Arrays.asList(surface, mImageReader.getSurface()),
-                    mSessionCallback, null);
+            if (mIsScanning) {
+                mPreviewRequestBuilder.addTarget(mScanImageReader.getSurface());
+            }
+            mCamera.createCaptureSession(Arrays.asList(surface, mStillImageReader.getSurface(),
```

Comment:
There is an output while executing this statement: 
> E/CameraCaptureSession: Session 0: Failed to create capture session; configuration failed

This log is print from the internal source code. I researched but not get any progress.
I tried `createCaptureSession` with a list of two surface bugs got over, but Then I can not use the both surface as output surface. 

So things become bad: I can only use one imagereader of the two, capture it or preview it.

Do you have any idea?